sdk/storage/Azure.Storage.Common/src/StorageClientOptions.cs

```diff
@@ -86,7 +86,8 @@ public static HttpPipelinePolicy GetAuthenticationPolicy(object credentials = nu
         public static HttpPipeline Build(this ClientOptions options, HttpPipelinePolicy authentication = null) =>
             HttpPipelineBuilder.Build(
                 options,
-                authentication);
+                authentication,
+                new StorageRequestValidationPipelinePolicy());
```

Comment:
We should probably have a static instance called `StorageRequestValidationPipelinePolicy.Shared` rather than create a new one of these for every pipeline.

Reply:
Do you mean just make a StorageRequestValidationPipleline object a member of the StorageClientOptions class and have it use that object every time it calls Build?

Reply:
Since all of these policies are stateless, one of the patterns we use is to have a singleton that we share to avoid allocations.  In the StorageRequestValidationPipelinePolicy class, you could add a `public static StorageRequestValidationPipelinePolicy Shared { get; } = new StorageRequestValidationPipelinePolicy();` and then at this spot just pass `StorageRequestValidationPipelinePolicy.Shared` instead of creating a new instance for every client.